relay-ourlogs/src/ourlog.rs

```diff
@@ -201,4 +206,61 @@ mod tests {
             Some(&"SELECT \"table\".\"col\" FROM \"table\" WHERE \"table\".\"col\" = %s".into())
         );
     }
+
+    #[test]
+    fn parse_log_without_observed_time() {
+        let json_without_observed_time = r#"{
+            "timeUnixNano": "1544712660300000000",
+            "observedTimeUnixNano": "0",
+            "severityNumber": 10,
+            "severityText": "Information",
+            "traceId": "5B8EFFF798038103D269B633813FC60C",
+            "spanId": "EEE19B7EC3C1B174",
+            "body": {
+                "stringValue": "Example log record"
+            },
+            "attributes": []
+        }"#;
+
+        let before_test = Utc::now().timestamp_nanos_opt().unwrap_or(0) as u64;
+        let otel_log: OtelLog = serde_json::from_str(json_without_observed_time).unwrap();
+        let our_log: OurLog = otel_to_sentry_log(otel_log);
+        let after_test = Utc::now().timestamp_nanos_opt().unwrap_or(0) as u64;
+
+        let observed_time = our_log.observed_timestamp_nanos.value().unwrap();
+        assert!(*observed_time > 0);
+        assert!(*observed_time >= before_test);
+        assert!(*observed_time <= after_test);
```

Comment:
I looked briefly but didn't see any prior examples of mocking `Utc::now()`, open to suggestions here.

Reply:
Unfortunately as good as it gets, our testing capatbilities for time are really limited at the moment.